packages/casting/src/follower-cosmjs.js

```diff
@@ -10,6 +10,15 @@ import { MAKE_DEFAULT_DECODER, MAKE_DEFAULT_UNSERIALIZER } from './defaults.js';
 import { makeCastingSpec } from './casting-spec.js';
 import { makeLeader as defaultMakeLeader } from './leader-netconfig.js';
 
+// A lot of cosmjs classes end up hardened through instances shared by this
+// package so preemptively harden them all.
+// However we cannot directly harden a module namespace object (exotic behavior
+// for bindings) so spread the namespace instead
+harden({
+  tendermint34: { ...tendermint34 },
+  stargateStar: { ...stargateStar },
+});
```

Comment:
Strange to construct a record that you then drop on the floor. If no one can use it, why bother?

Reply:
Because the properties contain values that I want hardened.

Reply:
```suggestion
harden([ ...tendermint34, ...stargateStar ]);
```

Reply:
I am hoping we get to fix the harden path diagnostics if there are any errors during hardening. What's the concern with a record vs an array?

Reply:
> ... path diagnostics ...

That makes sense! No change suggested.

Reply:
Resolving.